tests/unit-tests/test_bit_math.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+
+import sys
+from unittest.mock import MagicMock
+
+# Replace `pwndbg.commands` module with a mock to prevent import errors, as well
+# as the `load_commands` function
+module_name = "pwndbg.commands"
+module = MagicMock(__name__=module_name, load_commands=lambda: None)
+sys.modules[module_name] = module
```

Comment:
Why do we need this?

Reply:
Everyone is just copying and pasting the preamble from the other tests 😛 We should probably do this in a file called `mocks/commands.py` or something, so the user can choose to import it if they want. But that's out of scope for this PR.

Reply:
Oop - yep, just copied them from the other files. Removed them now.